recipes/boost-ext-simd/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class BoostExSimdConan(ConanFile):
+    name = "boost-ext-simd"
```

Comment:
Since this seems to be one of many forks, applying our pre-fixing naming rule might be better

It would be then something like `procxx-boost-ext-simd`

It is not talking about forks, but some principal:
https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#should-reference-names-use---or-_